angular/demo/src/app/samples/select/basic.route.ts

```diff
@@ -0,0 +1,40 @@
+import {AgnosUIAngularModule, provideWidgetsConfig} from '@agnos-ui/angular';
+import {CommonModule} from '@angular/common';
+import {Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+
+@Component({
+	standalone: true,
+	imports: [AgnosUIAngularModule, CommonModule, FormsModule],
+	providers: [provideWidgetsConfig()],
+	template: `
+		<div style="height: 400px;">
+			<h2>Basic examples</h2>
```

Comment:
Remove the h2 here (same in other frameworks), as the title of the sample is defined in the demo +page.svelte.